WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsUnknownView.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import SwiftUI
+
+final class CardReaderSettingsUnknownViewController: UIHostingController<CardReaderSettingsUnknownView>, CardReaderSettingsViewModelPresenter {
+    func configure(viewModel: CardReaderSettingsPresentedViewModel) {
+        guard let viewModel = viewModel as? CardReaderSettingsUnknownViewModel else {
+            DDLogError("Unexpectedly unable to downcast to CardReaderSettingsUnknownViewModel")
+            return
+        }
+        rootView.viewModel = viewModel
+    }
+
+    init(viewModel: CardReaderSettingsUnknownViewModel) {
+        super.init(rootView: CardReaderSettingsUnknownView(viewModel: viewModel))
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+struct CardReaderSettingsUnknownView: View {
+    @State var viewModel: CardReaderSettingsUnknownViewModel
+    @State var presentedURL: URL? = nil
+
+    var body: some View {
```

Comment:
I'd welcome programmatic definition of views (over storyboard). So much easier to understand diffs. I am curious if @ctarda concurs. I've used both approaches professionally. The big advantage of storyboards (IMHO) is designers can develop components, although that is not our approach here.

Reply:
> The big advantage of storyboards (IMHO) is designers can develop components

In my experience, it's been very rare that we were able to build a component using just what Storyboards offer. It might allow for easy tweaking of the visual properties once its build, but Xcode also offers some design tools to create and modify SwiftUI hierarchies without writing code.